### PR TITLE
integration/env: restore session token env variables

### PR DESCRIPTION
**Pull request by kopiczko**
This might be actually useful for local testing. No reason to always put
empty string there.

### CODE REVIEW

File: integration/env/aws.go [hunk 1]
```diff
@@ -1,5 +1,3 @@
-// +build k8srequired
-
```
Comment (xh3b4sd) [inline integration/env/aws.go]: Why removing this? What is the rule? I would like to keep this usefully consistent.

Comment (kopiczko) [reply integration/env/aws.go]: I just copied it from e2esetup.

File: integration/env/aws.go [hunk 2]
```diff
@@ -1,5 +1,3 @@
...
 package env
```
Comment (kopiczko) [inline integration/env/aws.go]:
```suggestion
// +build k8srequired

package env
```

File: integration/env/aws.go (hunk 1 above)
Comment (kopiczko) [reply integration/env/aws.go]: Restored.

**Pull request merged**